### DISCUSSION TITLE
Is storage for attachments and notes still limited to 1GB with VW?

### DISCUSSION THREAD

**Discussion by DudeThatsErin** [dani-garcia/vaultwarden#2770] [Q&A] [answered]
Title basically says it all. Just curious if it would be limited by my VPS I would host it on or by the software/code?

**Comment by bokkabonga** [comment, marked as answer]
You can configure the limits for orgs and users yourself. See https://github.com/dani-garcia/vaultwarden/blob/main/.env.template
What you are looking for are the values "ORG_ATTACHMENT_LIMIT" and "USER_ATTACHMENT_LIMIT"

I´m not sure about the defaults for those values if not configured though.

**Reply by DudeThatsErin** [reply]
Okay cool, thank you. I'll have to see if I can get this working tomorrow. If so, would push me back from 1Password.

**Reply by BlackDex** [reply]
By default there are no storage limits. You can configure them if you want though.

There is only a limit in regards to the upload size per attachment, since that is determined by the clients, which is about 500MiB.
